katsdpcontroller/test/test_product_controller.py

```diff
@@ -346,6 +346,24 @@ def _setup_band_mask_model(self) -> None:
             '/models/band_mask/fixed/test.h5'
         )
 
+    def _setup_primary_beam_model(self) -> None:
+        # Model is not used for anything, so do a minimal amount to get it working.
+        model = katsdpmodels.primary_beam.PrimaryBeamAperturePlane(
+            -1 * u.m, -1 * u.m,
+            1 * u.m, 1 * u.m,
+            [1, 2] * u.GHz,
+            np.zeros((2, 2, 2, 8, 8), np.complex64),
```

Comment:
Why 8x8? I only see -1 m start and 1 m step. Where does the 8 happen?

Reply:
It's the number of steps. It happens right here - it doesn't need to be repeated elsewhere.

Reply:
Ah, right, gotcha. It _might_ have been clearer if start = -3, step = 1 and number of steps = 7, suggesting a symmetrical grid on the aperture plane, but I don't think it's worth changing.

Reply:
Mattieu's models aren't symmetric either :-) Although less asymmetric (just one pixel) than this ended up being. As you say, not worth changing.